docs/it/aspetti-finanziari-i-soldi.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: default
+title: Aspetti Finanziari - "I SOLDI"
+nav_order: 44
+---
+
+<!-- prettier-ignore-start -->
+# Aspetti Finanziari "I Soldi"?
+{: .no_toc }
+
+- TOC
+{:toc}
+
+<!-- prettier-ignore-end -->
+
+Il settore delle tecnologie dell’informazione e della comunicazione, anche noto con gli acronimi ICT
```

Comment:
Trovo tutto il paragrafo poco scorrevole. Basterebbe aggiungere un paio di `.` in più secondo me (far diventare 2 frasi mozzafiato 4 frasi)